DA/submissions/pickled favorites

```diff
@@ -0,0 +1,9 @@
+import pickle
+favorites = ['football', 'wrestling', 'weightlifting', 'coding']
+f = open('favorites.dat', 'wb')
+pickle.dump(favorites, f)
+f.close()
+load_favourites= open('favorites.dat', 'rb')
```

Comment:
why are there two spellings of favorites lol. I'm all for the british english spelling but lets stick to one :)

Reply:
fixed